apps/nextjs/src/app/(main)/layout.tsx

```
@@ -57,12 +56,22 @@ export default function MainContentLayout({
-        <MainSection>{children}</MainSection>
-        {footer}
-      </div>
+    <div className="flex w-full flex-col items-center shrink-0 bg-neutral-950 shadow-md will-change-transform dark:shadow-white/10 lg:w-11/12 xl:w-10/12 2xl:w-8/12">
+      {navbar}
+
+      <main
+        id="main-content"
+        className="flex min-h-screen w-full flex-col items-center"
+      >
+        <section
+          role="region"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)